sosw/components/test/unit/test_helpers.py

```diff
@@ -810,5 +810,14 @@ def test_is_event_from_sqs__many(self):
                          unwrap_event_recursively(deepcopy(SQS_EVENT_MANY)))
 
 
+    def test_small_int_from_string(self):
+        self.assertEqual(small_int_from_string("hello world"), 91)
+        self.assertEqual(small_int_from_string("hello world", 3), 291)
+        self.assertEqual(small_int_from_string("sosw is the best", 1), 0)
+        self.assertEqual(small_int_from_string("sosw is the best", 5), 68280)
+
+        with self.assertRaises(ValueError):
+            small_int_from_string("test", num_digits=-1)
+
```

Comment:
Add running same assertion 3 times to guarantee that the result is reproducable.